soccer/Processor.hpp

```diff
@@ -99,7 +99,10 @@ class Processor : public QThread {
     bool joystickValid();
     JoystickControlValues getJoystickControlValues();
 
-    void externalReferee(bool value) { _externalReferee = value; }
+    void externalReferee(bool value) {
+        _externalReferee = value;
```

Comment:
`_externalReferee` dosen't seem used anywhere except the construtor of this class, maybe it could be removed (and anywhere it is needed could reference _refereeModule? That way we don't have multiple references to the same value.